### PR TITLE
chore(resource-detector-instana): use exported strings for attributes

### PR DESCRIPTION
**Pull request by maryliag**
## Which problem is this PR solving?

- Part Of #2025 

## Short description of the changes

On package `opentelemetry-resource-detector-instana`:
- Update `@opentelemetry/semantic-conventions` from `^1.0.0` to `^1.22.0`
- Use exported strings for Semantic Resource Attributes.

### CODE REVIEW

**Comment by JamieDanielson** [comment]
Just a note, I recently updated the [CONTRIBUTING doc](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/CONTRIBUTING.md#changelog) because we no longer need to update the changelog on our own in PRs - it gets updated in the releasing step!

File: detectors/node/opentelemetry-resource-detector-instana/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ import { NodeSDK } from "@opentelemetry/sdk-node";
 import { instanaAgentDetector } from "@opentelemetry/resource-detector-instana";
 
 const globalResource = new Resource({
-   [SemanticResourceAttributes.SERVICE_NAME]: "TestService",
+   [SEMRESATTRS_SERVICE_NAME]: "TestService",
```
Comment (trentm) [inline detectors/node/opentelemetry-resource-detector-instana/README.md:34]: Perhaps update the `import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";` a couple lines up.

**Review by trentm — COMMENTED** [review]
Just the one README nit.

File: detectors/node/opentelemetry-resource-detector-instana/README.md (hunk 1 above)
Comment (maryliag) [reply detectors/node/opentelemetry-resource-detector-instana/README.md:34]: Great catch! Done

**Review by kirrg001 — APPROVED** [review]

**Review by JamieDanielson — APPROVED** [review]
Thanks for this @maryliag ! Just a note, it's helpful for reviewers if you actually do not force-push changes, and instead keep the history. Not a big deal on smaller PRs, but as changes are made it can be difficult to see what's changed.

**Review by trentm — APPROVED** [review]

**Pull request merged**